ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh

```diff
@@ -34,10 +34,12 @@ if [ -z "${UPGRADE_SKIP_TAGS}" ] ; then
 else
     export UPGRADE_SKIP_TAGS="${UPGRADE_SKIP_TAGS} and not @customer and not @security"
 fi
+UPGRADE_SKIP_TAGS_CLOUD="${UPGRADE_SKIP_TAGS/and not @destructive/}"
```

Comment:
Thanks, @liangxia 

How does `${UPGRADE_SKIP_TAGS/and not @destructive/}` translate?

Reply:
It removes `and not @destructive` from the default UPGRADE_SKIP_TAGS